### ISSUE TITLE
useRuntimeConfig is not defined

### ISSUE THREAD

**Issue by 8uff3r** [YsarocK/nuxt-appwrite#46]
**Describe the bug**
After enabling the server guard i get the following error:
```js
ERROR  [worker reload] [worker init] useRuntimeConfig is not defined    1:09:49 AM

  at .nuxt/dev/index.mjs:659:16
  at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
```

**To Reproduce**
1. Add the package to project.
2. Set the environment variables
3. Go to any route
4. See the error

**Expected behavior**
User should be able to see the pages

**Screenshots**
![Screenshot 2023-06-28 at 01-49-38 503 Server Unavailable](https://github.com/YsarocK/nuxt-appwrite/assets/95995027/db8c021d-a37a-4479-a182-442e72e5084b)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Firefox

**Comment by YsarocK** [comment]
Hi!

Thanks, I'll check tomorrow. I'll get you in touch asap.

**Comment by YsarocK** [comment]
Hi,

Should be resolved now in [1.6.3](https://github.com/YsarocK/nuxt-appwrite/releases/tag/v1.6.3)

Have a good day
